### ISSUE TITLE
Startup problems

### ISSUE THREAD

**Issue by guenti** [macvim-dev/macvim#424]
When I start mvim (cli or App starter) I see sometimes this, even when I try mvim -N -u NONE. Closing MacVim and restart helps sometimes.

![bildschirmfoto 2016-11-30 um 17 42 25](https://cloud.githubusercontent.com/assets/125518/20761975/09c5ccd6-b725-11e6-91ec-844f81322681.png)

OS: macOS 10.12.1 (16B2555)

**Comment by splhack** [comment]
Can you try snapshot-118 https://github.com/macvim-dev/macvim/releases/tag/snapshot-118 with the user defaults setting?

    $ defaults write org.vim.MacVim MMUseCGLayerAlways -bool YES

**Comment by guenti** [comment]
Sure.

If I fire up mvim this error came up:
Fatal Python error: Py_Initialize: unable to load the file system codec
ImportError: No module named 'encodings'

Current thread 0x00007fffd39d23c0 (most recent call first):
Vim: Caught deadly signal ABRT
Vim: Finished.

If I start MacVim via App starter the menu exists, but no window at all, even if I select New Window from the menu.

Python Versions are the standard system Python coming with macOS 10.12 and Python 3.5.2 from python.org.

This error cames up the first time. 'til snapshot-117 that was not happen.

**Comment by splhack** [comment]
Thank you for the information. snapshot-119 should be fine with python.org 3.5.2 binary
https://github.com/macvim-dev/macvim/releases/tag/snapshot-119

**Comment by guenti** [comment]
Same behaviour with snapshot-119. See the pic.
![bildschirmfoto 2016-12-09 um 14 06 27](https://cloud.githubusercontent.com/assets/125518/21050067/db991d72-be18-11e6-9d24-b7b7c117f1a3.png)

**Comment by splhack** [comment]
@guenti what does your defaults look like?

    $ defaults read org.vim.MacVim

**Comment by guenti** [comment]
The output is:

    MMAutosaveColumns = 134;
    MMAutosaveRows = 38;
    MMTopLeftPoint = "{1852, 669}";
    MMUseCGLayerAlways = 1;
    NSQuotedKeystrokeBinding = "";
    NSRepeatCountBinding = "";
    SUEnableAutomaticChecks = 0;
    SUHasLaunchedBefore = 1;
    SUSendProfileInfo = 0;

**Comment by guenti** [comment]
Hi there!

I have an external monitor on my MBP and the behaviour only exists if MacVim opens on the external monitor. Weird.

**Comment by settermjd** [comment]
I have that behaviour regardless of where I'm trying to use MacVim. :-(

**Comment by magiclyx** [comment]
Hi，I have the same issue. Is there a solution now?

**Comment by magiclyx** [comment]
Hi, snapshot-119 solved the problem . But the problem reappears in 144 version.

**Comment by magiclyx** [comment]
Hi, snapshot-119 solved the problem, but it appears again in snapshot-144.

**Comment by jschloer** [comment]
I am also getting the " No module named 'encodings'"crash that guenti mentioned, but only when I start editing and only after upgrading this morning. Let me know if there's any information I can gather to help debug.

**Comment by eirnym** [comment]
@guenti  Is it still a problem for you?  
@jschloer Is it still a problem for you?

**Comment by jschloer** [comment]
Just updated to the latest to be sure and it works for me. Thanks.

**Comment by guenti** [comment]
No, it works with snapshot-155.

**Comment by ychin** [comment]
Thanks for following up @eirnym. I will close it out. File another issue if it happens again.
